apps/ledger-live-mobile/src/screens/PTX/Earn/EarnInfoDrawer.tsx

```diff
@@ -1,12 +1,17 @@
+import { Button, Flex, Icons, Link, Text } from "@ledgerhq/native-ui";
 import React, { useCallback, useEffect, useState } from "react";
-import { Button, Flex, Text } from "@ledgerhq/native-ui";
 import { useTranslation } from "react-i18next";
 
+import InformationFill from "@ledgerhq/icons-ui/native/InformationFill";
```

Comment:
I'm amazed that `<Icons.InformationFill ... />` works even though you use the sub-module import instead of all of `Icons`!

Reply:
I had this as an extra unused imports. Replaced it with Icons.InformationFill since that seems to be the norm?